i18n/locale-en.json

```diff
@@ -0,0 +1,265 @@
+{
+  "I18N_GALLERY_PAGE_TITLE": "Oppia - Gallery",
+  "I18N_GALLERY_TITLE": "Hi, I'm Oppia",
+  "I18N_GALLERY_SUBTITLE": "Your personal tutor for ",
+  "I18N_GALLERY_SUBTITLE_COLOR_anything": "anything",
+  "I18N_GALLERY_CREATE_EXPLORATION": "Create exploration",
+  "I18N_GALLERY_UPLOAD_EXPLORATION": "Upload exploration",
+  "I18N_GALLERY_SEARCH_PLACEHOLDER": "What are you curious about?",
+  "I18N_GALLERY_ALL_CATEGORIES": "All Categories",
+  "I18N_GALLERY_ALL_CATEGORIES_SELECTED": "All categories selected",
+  "I18N_GALLERY_CATEGORIES_ARCHITECTURE": "Architecture",
+  "I18N_GALLERY_CATEGORIES_ART": "Art",
+  "I18N_GALLERY_CATEGORIES_BIOLOGY": "Biology",
+  "I18N_GALLERY_CATEGORIES_BUSINESS": "Business",
+  "I18N_GALLERY_CATEGORIES_CHEMISTRY": "Chemistry",
+  "I18N_GALLERY_CATEGORIES_CODING": "Coding",
+  "I18N_GALLERY_CATEGORIES_COMPUTING": "Computing",
+  "I18N_GALLERY_CATEGORIES_ECONOMICS": "Economics",
+  "I18N_GALLERY_CATEGORIES_EDUCATION": "Education",
+  "I18N_GALLERY_CATEGORIES_ENGINEERING": "Engineering",
+  "I18N_GALLERY_CATEGORIES_ENVIRONMENT": "Environment",
+  "I18N_GALLERY_CATEGORIES_GEOGRAPHY": "Geography",
+  "I18N_GALLERY_CATEGORIES_GOVERNMENT": "Government",
+  "I18N_GALLERY_CATEGORIES_HOBBIES": "Hobbies",
+  "I18N_GALLERY_CATEGORIES_INTERACTIVE_FICTION": "Interactive Fiction",
+  "I18N_GALLERY_CATEGORIES_LANGUAGES": "Languages",
+  "I18N_GALLERY_CATEGORIES_LAW": "Law",
+  "I18N_GALLERY_CATEGORIES_LIFE_SKILLS": "Life Skills",
+  "I18N_GALLERY_CATEGORIES_MATHS": "Maths",
+  "I18N_GALLERY_CATEGORIES_MATHEMATICS": "Mathematics",
+  "I18N_GALLERY_CATEGORIES_MEDICINE": "Medicine",
+  "I18N_GALLERY_CATEGORIES_MUSIC": "Music",
+  "I18N_GALLERY_CATEGORIES_PHILOSOPHY": "Philosophy",
+  "I18N_GALLERY_CATEGORIES_PHYSICS": "Physics",
+  "I18N_GALLERY_CATEGORIES_PROGRAMMING": "Programming",
+  "I18N_GALLERY_CATEGORIES_PSYCHOLOGY": "Psychology",
+  "I18N_GALLERY_CATEGORIES_PUZZLES": "Puzzles",
+  "I18N_GALLERY_CATEGORIES_READING": "Reading",
+  "I18N_GALLERY_CATEGORIES_RELIGION": "Religion",
+  "I18N_GALLERY_CATEGORIES_SPORT": "Sport",
+  "I18N_GALLERY_CATEGORIES_STATISTICS": "Statistics",
+  "I18N_GALLERY_CATEGORIES_TEST": "Test",
+  "I18N_GALLERY_CATEGORIES_WELCOME": "Welcome",
+  "I18N_GALLERY_N_CATEGORIES": "<[categoriesCount]> Categories",
+  "I18N_GALLERY_ALL_LANGUAGES": "All Languages",
+  "I18N_GALLERY_ALL_LANGUAGES_SELECTED": "All languages selected",
+  "I18N_GALLERY_LANGUAGES_EN": "English",
```

Comment:
If possible, take autonyms from CLDR instead of adding them to your locale files.